### PR TITLE
dandi-schema readme edit

### PR DESCRIPTION
**Pull request by melster1010**
Edit readme for stylesheet compliance/grammar/clarity/consistency.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,23 +1,36 @@
-A Python library for maintaining and managing DANDI metadata schemata. The
-library helps create and validate DANDI schema-compliant metadata for Dandisets
-and assets.
+# DANDI Schema
 
-To use: `pip install dandischema`
+The dandi-schema repository is a Python library for maintaining and managing DANDI metadata schemata.
```
Comment (waxlamp) [inline README.md:3]:
```suggestion
`dandi-schema` is a Python library for maintaining and managing DANDI metadata schemata. 
```

File: README.md [hunk 2]
```diff
@@ -1,23 +1,36 @@
...
-the DANDI API.
+## Installation
 
-This library uses [Pydantic](https://github.com/samuelcolvin/pydantic) to implement
+`pip install dandischema`
+
+## Description
+
+Every `Dandiset` and associated asset has a metadata object that can be retrieved using
+the DANDI API. This library helps create and validate DANDI schema-compliant metadata for `Dandisets`
```
Comment (waxlamp) [inline README.md:12]:
```suggestion
the DANDI API. This library helps create and validate DANDI schema-compliant metadata for Dandisets
```

File: README.md [hunk 3]
```diff
@@ -1,23 +1,36 @@
...
+and assets. It uses [Pydantic](https://github.com/samuelcolvin/pydantic) to implement
 all the metadata classes. Schemas are generated on schema modifications and placed into
-[this repository](https://github.com/dandi/schema/tree/master/releases).
+[this repository](https://github.com/dandi/schema/tree/master/releases).
```
Comment (waxlamp) [inline README.md:15]:
```suggestion
[this repository](https://github.com/dandi/schema/tree/master/releases).
```

File: README.md [hunk 4]
```diff
@@ -1,23 +1,36 @@
...
+
+Every `Dandiset` and associated asset has a metadata object that can be retrieved using
+the DANDI API. This library helps create and validate DANDI schema-compliant metadata for `Dandisets`
+and assets. It uses [Pydantic](https://github.com/samuelcolvin/pydantic) to implement
 all the metadata classes. Schemas are generated on schema modifications and placed into
-[this repository](https://github.com/dandi/schema/tree/master/releases).
+[this repository](https://github.com/dandi/schema/tree/master/releases). 
 
-Dandischema generates JSON schema definitions and also an associated `context.json`
+Dandi-schema generates JSON schema definitions and also an associated `context.json`
```
Comment (waxlamp) [inline README.md:17]:
```suggestion
`dandi-schema` generates JSON schema definitions and also an associated `context.json`
```

File: README.md [hunk 5]
```diff
@@ -1,23 +1,36 @@
...
 file for JSON-LD compliance of the metadata models.
 
+Important files in this repository include:
 - models.py - contains the models and any changes should be made there
```
Comment (waxlamp) [inline README.md:21]: `any changes should be made there` seems like an odd thing to include in the description of the file. This is presumably *not* the exclusive place to make changes in this repository. Not sure what to do about the phrasing in this PR (maybe nothing) but just pointing it out nonetheless.

File: README.md [hunk 6]
```diff
@@ -1,23 +1,36 @@
...
 - metadata.py - contains functions for validating, migrating, and aggregating metadata
 - datacite.py - converts the Dandiset metadata to a Datacite metadata structure
 
 The generated JSON schemas can be used together with
 [VJSF](https://koumoul-dev.github.io/vuetify-jsonschema-form/latest/) to create a UI
-for metadata modification. The DANDI Web app uses this for Dandiset metadata modification.
+for metadata modification. The DANDI Web app uses this to modify `Dandiset` metadata.
```
Comment (waxlamp) [inline README.md:27]:
```suggestion
for metadata modification. The DANDI Web app uses this to modify Dandiset metadata.
```

File: README.md [hunk 7]
```diff
@@ -1,23 +1,36 @@
...
+
+## Resources
+
+* To learn how to interact with the DANDI archive,
+see [the handbook](https://www.dandiarchive.org/handbook/).
+
+* To file a feature request or bug report, go to https://github.com/dandi/helpdesk/issues/new/choose.
+
+* For all other issues, contact the DANDI team: help@dandiarchive.org.
```
Comment (waxlamp) [inline README.md:36]: I think removing the blank lines results in a nicer rendering of bulleted lists.

```suggestion
* To learn how to interact with the DANDI archive,
see [the handbook](https://www.dandiarchive.org/handbook/).
* To file a feature request or bug report, go to https://github.com/dandi/helpdesk/issues/new/choose.
* For all other issues, contact the DANDI team: help@dandiarchive.org.
```

**Review by waxlamp — CHANGES_REQUESTED** [review]
Text looks good. My suggestions are mostly to do with formatting/style.

File: README.md (hunk 5 above)
Comment (melster1010) [reply README.md:21]: revised

File: README.md (hunk 7 above)
Comment (melster1010) [reply README.md:36]: agreed

File: README.md (hunk 6 above)
Comment (melster1010) [reply README.md:27]: (see comment for line 12)

File: README.md (hunk 2 above)
Comment (waxlamp) [reply README.md:12]: Then I agree that, for consistency, it should be formatted as a code element.

But on a broader level, I don't think that is the right choice. Can you point me to the style policy/discussion where this decision was made?

**Review by waxlamp — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (waxlamp) [reply README.md:12]: Let's make the decision now to remove the code element from the term "Dandiset". Can you update the stylesheet, @melster1010?

Comment (melster1010) [reply README.md:12]: @waxlamp sure - I'll have to update the stylesheet, remove the special formatting from "Dandiset" in the Handbook files, as well as this readme (as separate PR)

**Pull request merged**